components/intl/__tests__/DateFormatter-test.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import DateFormatter, { DateFormatterProps } from '../DateFormatter';
+
+it('should render correctly', () => {
+  expect(shallowRender()).toMatchSnapshot('standard');
+  expect(shallowRender({ long: true })).toMatchSnapshot('long');
+});
+
+function shallowRender(overrides: Partial<DateFormatterProps> = {}) {
```

Comment:
Small tip: you don't need to export the props of functional component for testing anymore, you can do `Partial<typeof DateFormatter>` instead

Reply:
I might be missing something, but wouldn't `CreationDateFacet` be "simply" fixed with:
```
const tooltip = (
        // eslint-disable-next-line react/jsx-fragments
        <React.Fragment>
          {formatMeasure(stats[start], 'SHORT_INT')}
          <br />
          <DateFormatter date={startDate} long={true} />
          {!isSameDay(tooltipEndDate, startDate) && (
            <DateFormatter date={tooltipEndDate} long={true}>
              {formatted => ` - ${formatted}`}
            </DateFormatter>
          )}
        </React.Fragment>
      );
```

Reply:
No idea I didn't dig into it ^^